cmd/device-worker/main.go

```diff
@@ -20,7 +20,8 @@ import (
 	os2 "github.com/project-flotta/flotta-device-worker/internal/os"
 	registration2 "github.com/project-flotta/flotta-device-worker/internal/registration"
 	"github.com/project-flotta/flotta-device-worker/internal/server"
-	workload2 "github.com/project-flotta/flotta-device-worker/internal/workload"
+	"github.com/project-flotta/flotta-device-worker/internal/service"
+	workload "github.com/project-flotta/flotta-device-worker/internal/workload"
```

Comment:
this workload alias is not needed at all, no?

Reply:
yeah, will remove it.